### ISSUE TITLE
Callback termination for PILOT/FAILED

### ISSUE THREAD

**Issue by lee212** [radical-cybertools/radical.entk#397]
When a job is exited by a scheduler i.e. LSF, PILOT changes a state to `FAILED` which requires termination of EnTK. This is directly related to #389 and possibly #393.
```
   2019-12-12 11:00:53,262: umgr.0000           : 
task-manager                    : 
umgr.0000.subscriber._state_sub_cb: DEBUG   : 
umgr state cb ignores pilot.0000/FAILED
 2019-12-12 11:00:53,801: umgr.0000           : 
task-manager                    :
umgr.0000.idler._unit_pull_cb: INFO    : units pulled:    0
```

**Comment by lee212** [comment]
@andre-merzky , I am devoting more time to this as this may block Eugen's project and asking your help to resolve this together if possible.

**Comment by andre-merzky** [comment]
@lee212 : Thanks for the ping, happy to look into it.  Does that happen with the example EnTK scripts, too, or only with a specific use case?  Is that only on Summit, or does the same happen in your local env?  
I'll try to reproduce it, but am afraid I will only be online for some hours tomorrow before going into the weekend.  Lets see how far we get...

**Comment by lee212** [comment]
It might be fixed with the commit 215d206f5447a82be8da20c02bb68713841f35a2 . Eugen will give it a try. FYI, it happens when a task runs over given walltime. `sleep n` seconds for walltime m (n > m) is a simple example I tested and failed. This was observed on Summit but I can try on local env, I think it's a good idea. Thank you for the help!

**Comment by andre-merzky** [comment]
Was this resolved with 215d206 ?

**Comment by lee212** [comment]
Yes. I will close this
